x-pack/plugins/endpoint/public/applications/endpoint/store/index.ts

```diff
@@ -48,25 +48,30 @@ export const substateMiddlewareFactory = <Substate>(
   };
 };
 
-export const appStoreFactory = (coreStart: CoreStart): Store => {
+export const appStoreFactory: (coreStart: CoreStart, disableMiddleware?: boolean) => Store = (
+  coreStart,
+  disableMiddleware = false
+) => {
   const store = createStore(
     appReducer,
-    composeWithReduxDevTools(
-      applyMiddleware(
-        substateMiddlewareFactory(
-          globalState => globalState.managementList,
-          managementMiddlewareFactory(coreStart)
-        ),
-        substateMiddlewareFactory(
-          globalState => globalState.policyList,
-          policyListMiddlewareFactory(coreStart)
-        ),
-        substateMiddlewareFactory(
-          globalState => globalState.alertList,
-          alertMiddlewareFactory(coreStart)
+    disableMiddleware
```

Comment:
maybe put a comment in here saying why you need the ability to disable the middleware?

Reply:
durp